packages/webpack-env-define-plugin/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
You should add yourself as a contributor to the package.json. See http://browsenpm.org/package.json#contributors for details.